Makefile

```diff
@@ -35,6 +35,14 @@ endif
 wal-check: temp-install
 	$(prove_check)
 
+check: isolationcheck
+
+submake-isolation:
+	$(MAKE) -C $(top_builddir)/src/test/isolation all
+submake-rum:
```

Comment:
It will be good to move `submake-rum` target lower, closer to the `isolationcheck` target.